Source/Demos/Demo.Particles/Game1.cs

```diff
@@ -28,6 +28,7 @@ public Game1()
             Content.RootDirectory = "Content";
             IsMouseVisible = true;
             Window.AllowUserResizing = true;
+            Window.Position = Point.Zero;
```

Comment:
I'll double check by upgrading parallels, but I think there is problems with retina screens and MonoGame. I know prime31 was having issues with retina screens and MonoGame as well.